opal/corelib/runtime.js

```diff
@@ -1177,6 +1177,8 @@
 
   // The Array of ancestors for a given module/class
   Opal.ancestors = function(module) {
+    if (!module) { return []; }
```

Comment:
Why is it necessary?

Reply:
This is guarding a condition where module parameter is passed as nil.
This may be related to a bug, where the wrong context object passed around. Did not yet have time to investigate further. This seems save 'fix' for now and shouldn't hurt anything else.